src/cli/MeasureUpdateCommand.cpp

```diff
@@ -88,6 +97,203 @@ namespace cli {
+        ->required(false)
+        ->check(CLI::IsMember(taxonomyTags))
+        ->capture_default_str();
+      newMeasureSubCommand->add_option("-d,--description", opt->newMeasureOpts.description, "Description")->option_text("desc")->required(false);
+      newMeasureSubCommand->add_option("-m,--modeler-description", opt->newMeasureOpts.modelerDescription, "Modeler Description")
+        ->option_text("modeler_desc")
+        ->required(false);
+
+      [[maybe_unused]] auto* directoryPathOpt =
+        newMeasureSubCommand->add_option("DIRECTORY", opt->newMeasureOpts.directoryPath, "Directory for the measure")
+          ->option_text(" ")
+          ->required(true)
+          ->check(CLI::NonexistentPath);
+
+      {
+        static constexpr auto helpOptionsGroupName = "Help";
+        static constexpr auto newMeasureExamples = R"(Examples:
+
+1. Create a Ruby ModelMeasure:
+
+```
+$ openstudio labs measure new --class-name MyExampleRubyModelMeasure
+$ openstudio labs measure new --class-name MyExampleRubyModelMeasure --type ModelMeasure --language Ruby
+```
+
+2. Pass all optional args to create a Python EnergyPlusMeasure:
+
+```
+$ openstudio labs measure new --class-name MyExamplePythonMeasure --type EnergyPlusMeasure --language Python --name "My Python Measure" --description "This is my measure" --modeler-description "This does complicated stuff" --taxonomy-tag "Envelope.Form" ./test_measure
+
+$ openstudio labs measure new -c MyExamplePythonMeasure -t EnergyPlusMeasure -l Python -n "My Python Measure" -d "This is my measure" -m "This does complicated stuff" --taxonomy-tag "Envelope.Form" ./test_measure
+```
+
+3. List taxonomy tags
+```
+$ openstudio labs measure new --list-taxonomy-tags
+$ openstudio labs measure new --list-for-first-taxonomy-tag HVAC
+```
+)";
+        newMeasureSubCommand->set_help_flag("-h,--help", "Print this help message and exit")->group(helpOptionsGroupName);
+
+        newMeasureSubCommand
+          ->add_flag_callback(
+            "--examples",
+            []() {
+              fmt::print("{}\n", newMeasureExamples);
+              std::exit(0);  // NOLINT(concurrency-mt-unsafe)
+            },
+            "Show Example usage")
+          ->group(helpOptionsGroupName);
+
+        [[maybe_unused]] auto* listTaxonomyFlag = newMeasureSubCommand
+                                                    ->add_flag_callback(
+                                                      "--list-taxonomy-tags",
+                                                      [taxonomyTags]() {
+                                                        fmt::print("{}\n", taxonomyTags);
+                                                        std::exit(0);  // NOLINT(concurrency-mt-unsafe)
+                                                      },
+                                                      "List all available Taxonomy tags")
+                                                    ->group(helpOptionsGroupName);
```

Comment:
Couple of flags for taxonomy lookup, because even though I could just try to pass `openstudio measure new --taxonomy-tag Badval ./test_measure` and look at the error message to get accepted values, this is clearer to use `openstudio measure new --list-taxonomy-tags`

Also added one that looks up from a first level tag: `openstudio measure new --list-for-first-taxonomy-tag Envelope` (and throws if that first tag isn't valid)

**NOTE**: maybe this should live onto the `measure` subcommand and not the `measure new` subsubcommand, but that's maybe nitpicking, and it would reduce the readability of the help message in `measure` subcomand